src/components/containerGradient2/ContainerGradient.jsx

```diff
@@ -0,0 +1,266 @@
+import { useState, useEffect, useCallback } from 'react';
+import { Link } from 'react-router-dom';
+import { Transition } from 'react-transition-group';
+
+import styles from './ContainerGradient.module.scss';
+import { trimString } from '../../utils/utils';
+
+const classNames = require('classnames');
+
+function ContainerLamp({ style, children }) {
+  return (
+    <div
+      className={classNames(styles.wrapContainerLamp, {
+        [styles.wrapContainerLampGreen]: style === 'green',
+        [styles.wrapContainerLampBlue]: style === 'blue' || style === 'grey',
+        [styles.wrapContainerLampRed]: style === 'red',
+        [styles.wrapContainerLampYellow]: style === 'yellow',
+        [styles.wrapContainerLampPurple]: style === 'purple',
+        [styles.wrapContainerLampPink]: style === 'pink',
+        [styles.wrapContainerLampDefault]: !style,
+      })}
+    >
+      {children}
+    </div>
+  );
+}
+
+function ContainerLampAfter({ style, children }) {
+  return (
+    <div
+      className={classNames(styles.wrapContainerLampAfter, {
+        [styles.wrapContainerLampAfterGreen]: style === 'green',
+        [styles.wrapContainerLampAfterBlue]: style === 'blue',
+        [styles.wrapContainerLampAfterRed]: style === 'red',
+        [styles.wrapContainerLampAfterYellow]: style === 'yellow',
+        [styles.wrapContainerLampAfterPurple]: style === 'purple',
+        [styles.wrapContainerLampAfterDefault]: !style,
+      })}
+    >
+      {children}
+    </div>
+  );
+}
+
+function ContainerLampBefore({ style, children }) {
+  return (
+    <div
+      className={classNames(styles.wrapContainerLampBefore, {
+        [styles.wrapContainerLampBeforeGreen]: style === 'green',
+        [styles.wrapContainerLampBeforeBlue]: style === 'blue',
+        [styles.wrapContainerLampBeforeRed]: style === 'red',
+        [styles.wrapContainerLampBeforeYellow]: style === 'yellow',
+        [styles.wrapContainerLampBeforePurple]: style === 'purple',
+        [styles.wrapContainerLampBeforeDefault]: !style,
+      })}
+    >
+      {children}
+    </div>
+  );
+}
+
+function TxsStatus({ data }) {
+  let style;
+  switch (data.status) {
```

Comment:
also hasmap better approach than swithc case: like colors[data.status]

Reply:
it is copied component, will be refactored next releases